packages/components/src/GridLayout/Grid.component.js

```diff
@@ -51,7 +52,7 @@ function Grid({
 			compactType={compactType}
 			isResizable={isResizable}
 			useCSSTransforms={false}
-			verticalCompact={false}
+			verticalCompact={verticalCompact}
```

Comment:
You could use shortcut I guess
Dunno if Prettier will not doing it for you in fact
```suggestion
			verticalCompact
```